### ISSUE TITLE
Lag from excessive items in containers [$10]

### ISSUE THREAD

**Issue by Ochmar** [TwistedScorpio/OTHire#219]
Same problem faced here:
https://github.com/otland/forgottenserver/issues/1150
Sorry, the bug was caused by excessive items in the mail box ... if you can limit it would be great. The player was with some 70 thousand label in the mail box when the player entered the server was with lag.

**Comment by OchmanM** [comment]
It is indeed. But setting max limit is not solution, it's just workaround.
Imagine people trying to send huge lootbag via parcel or just lost their house, they need to access more than 2000 items to depot, even tho they cannot push there anything manually. And the house thing is the very likely scenario when it happens.

**Comment by Ochmar** [comment]
It is indeed. But setting max limit is not solution, it's just workaround.
Imagine people trying to send huge lootbag via parcel or just lost their house, they need to access more than 2000 items to depot, even tho they cannot push there anything manually. And the house thing is the very likely scenario when it happens.

**Comment by Anastaciaa** [comment]
What if we block sending parcels when dp is full, leaving houses too and disable house rent so ppls won't lose their houses?

**Comment by Ochmar** [comment]
You can do it if you can.
But it's a workaround, **not a solution**.

**Comment by Anastaciaa** [comment]
Yes i know, my friend working on asynchronous save on logout, so i just wondering what to do if he won't finish it on time.

**Comment by Ochmar** [comment]
I'm changing bounty from 10$ to 50$.

**Comment by Ochmar** [comment]
No longer bounty since Bountysource changed their ToS

**Comment by waqmaz93** [comment]
> No longer bounty since Bountysource changed their ToS

what the hell is ToS?
